examples/esp32c6/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2023-11-14"
```

Comment:
Perhaps this can be something a bit more modern, as indicated by previous comment `stable` should work nowadays?